### ISSUE TITLE
NVDA is announcing irrelevant information as checked over the 'New' menu item.

### ISSUE THREAD

**Issue by huangzumeihan** [microsoft/fluentui#20307]
issues link:https://developer.microsoft.com/en-us/fluentui#/controls/web/commandbar
Actual:
 NVDA is announcing irrelevant information as checked over the 'New ' menu item.
NVDA  announce as Email message  not checked  Email message 1 of 2

Expected:
 NVDA should not announce  irrelevant information as checked over the 'New  ' menu item.
NVDA should announce as -Email message 1 of 2
![image](https://user-images.githubusercontent.com/31198399/138407190-119b9225-b448-4753-98e4-86a956c61592.png)

**Comment by khmakoto** [comment]
Hi @huangzumeihan I just checked the link you sent with NVDA and I didn't get the irrelevant information you are mentioning. Can you check again to see if this is still an issue in the latest version of the library?
